src/AzureExtension/Strings/en-US/Resources.resw

```diff
@@ -26,36 +26,36 @@
         <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
         <comment>This is a comment</comment>
     </data>
-
-    There are any number of "resheader" rows that contain simple
+                

```

Comment:
Can you revert the space changes in this comment? I feel like some other tool will come along later and complain about trailing whitespace.